packages/insync-deb/insync-deb.pacscript

```diff
@@ -1,8 +1,16 @@
 name="insync-deb"
 pkgname="insync"
-version="3.5.4.50130"
-url="https://d2t3ff60b2tol4.cloudfront.net/builds/${pkgname}_${version}-focal_amd64.deb"
+version="3.7.6.50356"
+repology=("project: insync")
+codename="focal"
+hash="428be8951c5dd4b35d68edbf029f94b05dd10e3452c49ecdc819d44fdeca0f6d"
+echo reading hash
+if [[ "$(lsb_release -is)" == "Debian" ]]; then
+  codename="buster"
+  hash="baf1b3c424171d092053c2dd667fa0f167db5a37cd0a94b5eac1bc78d412a1a0"
+  echo this is debian
```

Comment:
Remove this line

Reply:
Yes sorry, this is a draft. I guess I should get a debian vm to try things out